### ISSUE TITLE
reassigning batches

### ISSUE THREAD

**Issue by vondele** [glinscott/fishtest#144]
the current reassignment of batches to new workers in order to reach 1000 games can lead to confusing result, e.g. game crashes of one worker can appear as crashes affecting other workers. See some discussion here:  https://github.com/official-stockfish/Stockfish/issues/1433

Similar observations have been made in the past (timeouts, high residuals, etc).

**Comment by Stefano80** [comment]
Ok, can we postpone this @vondele given the more pressing issues in the moment?

**Comment by vondele** [comment]
@Stefano80 sure... I greatly appreciate the effort put in these things, and I'm well aware it is a lot in the context of a hobby project.

**Comment by tomtor** [comment]
No, it is not. We still do that. Note that not completing partial tasks introduces issues because internally  the number of to be played games and the priority calculation depends on this.
